### DISCUSSION TITLE
Facturas à crédito

### DISCUSSION THREAD

**Discussion by ithustle** [assoft-portugal/SAF-T-AO#241] [Q&A] [answered]
Saudações,

O SAF-T AO, prevê lidar com facturas à crédito? Ou mais "angolanamente" falando, facturas feito a kilape? Se sim, poderia, por favor adicionar um exemplo e no mesmo incluir o pagamento da mesma.

Contextualizando, é feita uma factura à crédito em um mês e no mês seguinte essa factura é paga!!

**Comment by cryptolopes** [comment, marked as answer]
Obviamente que sim. A estrutura é exatamente igual à de qualquer outra fatura. No limite pode ser adicionada a data de vencimento através do elemento `PaymentTerms` https://github.com/assoft-portugal/SAF-T-AO/blob/a99ee01d5dd93415e96b4dd3d61224c8394ce301/XSD/SAFTAO1.01_01.xsd#L1073.

**Reply by ithustle** [reply]
@cryptolopes, obrigado por responder. Percebo. Alguma documentação disponível para ver a implementação, com exemplos e casos práticos?
